app/twitter/TwitterClient.py

```diff
@@ -303,11 +300,9 @@ def process_mentions(self, conversations, filter_spam=True):
         """Process mentions and add them to conversations"""
         print("\n=== Fetching Mentions ===")
 
-        last_mention_time = self.storage.get_timestamp("last_mention_time")
         mentions = self.client.get_users_mentions(
-            id=self.storage.get("user_id"),
-            max_results=50,
-            start_time=last_mention_time,
+            id=self.get_own_user_id(),
+            max_results=100,
```

Comment:
Let's limit this to 20 for now. We can increase it later. 

This might count towards API limits.